conf/shiro.ini.template

```diff
@@ -14,6 +14,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+# Zeppelin uses Shiro to configure Security. The first step of security is Authentication.
+# To enable authentication, you need to configure authc as the authentication method in 
+# the URL section of shiro.ini. Make sure the shiro.ini has "/** = authc" line uncommented
```

Comment:
It's just my opinion, but how about adding this info in here? 
"You can activate Shiro by copying `shiro.ini.template` to `shiro.ini.`" :)